definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -16,6 +16,8 @@ inputs:
         type: Directory
     star_fusion_genome_dir:
         type: Directory
+    cdna_file:
+        type: File
```

Comment:
I know that other parts of this pipeline still lack documentation, but would you consider using the "label" and "doc" lines to at least document your new changes? (see somatic_exome.cwl for examples)

Reply:
There's currently a mismatch between this input name and the name used as the input to the `check_strand` step.  I'd suggest renaming this one to `cdna_fasta` to match.